sql/create-db.sql

```diff
@@ -0,0 +1,76 @@
+-- Set up a PostgreSQL Database for use as the backing
```

Comment:
These seem correct to me. I've always been wary of triggers because somebody told me to be wary of them 10000 years ago. I don't have much empirical evidence for that now other than that they're kinda spooky-action-at-a-distance if you're just reading the code.

Do we have a plan for migrations and that sort of thing?